Added Mutated, CNA tables and issue fixes

kalletlak opened the pull request:
Fixed issues #46, #52, #59

kalletlak committed:
Added Mutated, CNA tables and issue fixes

Fixed issues #46, #52, #59

zhx828 commented on app/scripts/model/dataProxy.js:
```diff
@@ -254,12 +254,16 @@
+                          var _sampleIdArr = _ajaxPatient2SampleIdMappingObj[_tmpPatientId];
+                          _.each(_sampleIdArr, function (_sampleId) {
+                            // map from sample to patient
+                            _ajaxSample2PatientIdMappingObj[_sampleId] = new Array(_tmpPatientId);
```
Can we keep the consistency to use [_tmpPatientId] and avoid new Array or new Object?

kalletlak committed:
Updates to table view and chart header

1. Add/delete special charts.
2. Code cleanup

kalletlak committed:
Show selected genes in react table when the table in added

kalletlak committed:
Fix resetting special charts and code cleanup

kalletlak committed:
Some more changes in rendering tables

1. Support mutated genes and CNA tables for multi-study
2. Remove unwanted statements to improve performance

kalletlak committed:
Add gistic and mutsig images
